ts/client/src/client.ts

```diff
@@ -82,7 +82,7 @@ export class OpenBookV2Client {
     this.postSendTxCallback = opts?.postSendTxCallback;
     this.txConfirmationCommitment =
       opts?.txConfirmationCommitment ??
-      (this.program.provider as AnchorProvider).opts.commitment ??
+//      (this.program.provider as AnchorProvider).opts.commitment ??
```

Comment:
do we need this? if so, I will find a workaround

Reply:
I think it's nice to have it, because you might define default commitment behaviour on your provider. I know the nightmare of ts and anchor providers and wallets....